components/gitpod-protocol/src/permission.ts

```diff
@@ -6,18 +6,13 @@
 
 // see below for explanation
 export const Permissions = {
-    monitor: undefined,
     developer: undefined,
-    enforcement: undefined,
-    "privileged-ws": undefined,
     "registry-access": undefined,
     "admin-permissions": undefined,
     "admin-users": undefined,
     "admin-workspace-content": undefined,
     "admin-workspaces": undefined,
     "admin-projects": undefined,
-    "admin-api": undefined,
-    "ide-settings": undefined,
```

Comment:
Not sure how this was used.... @akosyakov You probably no? Still relevant somehow? :thinking:

Reply:
I think it was a feature flag to allow custom IDE docker images 2-3 years ago. It should not be relevant anymore.